core/chains/evm/client/node.go

```diff
@@ -338,6 +351,7 @@ func (n *node) Close() {
 		n.stateMu.Lock()
 		defer n.stateMu.Unlock()
 
+		n.cancelNodeCtx()
```

Comment:
This is the only place that `close(n.chStop)` is called, right? If so, I think nodeCtx can simply be a `context.WithCancel` and doesn't need a ContextFromChan

Reply:
Done